tests/addresslib/parser_mailbox_test.py

```diff
@@ -520,14 +515,10 @@ def test_local_part():
     run_mailbox_test('  steve@apple.com', 'steve@apple.com')
 
     # fail dot-atom
-    run_mailbox_test('steve @apple.com', None)
```

Comment:
These are valid addresses according to the RFC. Optional whitespace is allowed around dot-atoms.